### ISSUE TITLE
XEP-0363: HTTP File Upload

### ISSUE THREAD

**Issue by kruks23** [jappix/jappix#563]
Hi,

It would be nice if jappix implements XEP-0363: HTTP File Upload to share files:
http://xmpp.org/extensions/xep-0363.html

Sharing files in this way is very easy because you upload a file to your server and then you share the url.

Conversations and gajim implement this XEP.

**Comment by tbeitter** [comment]
very useful feature, indeed. Movim got it, too.

**Comment by tigre-bleu** [comment]
I agree, this is very usefull and widely used by all users I know using mobile devices (Conversations or Monal)
